libclambcc/ClamBCAnalyzer/ClamBCAnalyzer.cpp

```diff
@@ -125,61 +125,67 @@ bool ClamBCAnalyzer::runOnModule(Module &M)
         std::set<ConstantExpr *> ces;
         getDependentValues(gv, insts, globs, ces);
 
-        for (auto J : ces) {
-            ConstantExpr *CE = llvm::cast<ConstantExpr>(J);
-            // ClamAV bytecode doesn't support arbitrary constant expressions for
-            // globals, so introduce helper globals for nested constant expressions.
-            if (CE->getOpcode() != Instruction::GetElementPtr) {
-                if (CE->getOpcode() == Instruction::BitCast) {
-                    GlobalVariable *GV = new GlobalVariable(M, CE->getType(), true,
-                                                            GlobalValue::InternalLinkage,
-                                                            CE, I->getName() + "_bc");
-                    CEMap[CE]          = GV;
-                    continue;
+        /*It is necessary to add these twice, because there is a condition we
+         * can't use global idx 0 or 1 in the interpreter, since the size will
+         * be incorrect in the interpreter.  Look at line 2011 of bytecode.c
```

Comment:
line 2011 may change over time.  Instead, you could share a link to a line in a specific commit, like:

https://github.com/Cisco-Talos/clamav/blob/clamav-0.105.0/libclamav/bytecode.c#L2011

Although I think the line # already changed, because 2011 doesn't seem right.